packages/collaborativedrive/package.json

```
@@ -0,0 +1,55 @@
+{
+  "name": "@jupyter/collaborativedrive",
```

Maybe we should rename the package to the following, so it reads better, before publishing it?

```suggestion
  "name": "@jupyter/collaborative-drive",
```

---

Fine with me :+1: